python/cudf/cudf/tests/test_timedelta.py

```diff
@@ -908,7 +903,7 @@ def test_timedelta_index_ops_with_scalars(data, other_scalars, dtype, op):
         actual = other_scalars // gtdi
 
     if op == "floordiv":
-        # Hand-coding pytest.xfail behaviour for certain combinations
+        # Hand-coding xfail behaviour for certain combinations
```

Comment:
Let's not use xfail since it is easy to conflate pytest xfails
```suggestion
        # Expect exceptions for certain combinations
```

Reply:
Fixed by going to the applymarker approach